packages/snjs/lib/application.ts

```diff
@@ -1558,6 +1569,7 @@ export class SNApplication {
     (this.webSocketsService as unknown) = undefined;
     (this.settingsService as unknown) = undefined;
     (this.mfaService as unknown) = undefined;
+    (this.appVersion as unknown) = undefined;
```

Comment:
String value doesn't need to be de-inited manually. Only complex services which hang on to other objects.

Reply:
Noted, done 👍